eCourse_backend/templates/file_exchange/lecturer_exersice.html

```diff
@@ -0,0 +1,33 @@
+{% extends "lecturer/home_lecturer.html" %}
+{% load static %}
+
+{% block title %}Aufgabe{% endblock %}
+{% block site-title %}Aufgabe{% endblock %}
+
+{% block content %}
+  <div>
+    <h3>Aufgabe:</h3>
+    <iframe src="{% url 'upload_site' id=exersiceID %}" frameborder="0"></iframe>
+    <script src="{% static 'JS/iframe_size.js' %}"></script><br>
+    <h4>Hochgeladenes:</h4>
```

Comment:
is this a real German word? Sounds bad. Maybe Hochgeladene Dateien would be a better option.